Makefile.am

```diff
@@ -244,6 +265,7 @@ EXTRA_DIST += \
 	snapcraft/helpers \
 	snapcraft/snap \
 	babeld/Makefile \
+	mgmtd/Makefile
```

Comment:
Missing the \ at the end of the line.
This causes to the 'make dist' to generate incomplete files and breaks the package build on the RPM based systems on the CI

Reply:
@mwinter-osr Thanks a lot for pointing that out. Let me fix it asap. Assuming centos8 and fedora29 build will be successful after that, we still need to discuss the centos7 case.

Reply:
Ok. I have pushed the fix. Hopefully the centos8 and fedora29 builds will not fail again.